manim/mobject/number_line.py

```diff
@@ -68,6 +68,29 @@ class NumberLine(Line):
     kwargs : Any
         Additional arguments to be passed to :class:`~.Line`.
 
+    Examples
+    --------
+    .. manim:: NumLineExample
+
```

Comment:
```suggestion
        :save_last_frame:
```

Reply:
Suggested changes are included in the latest commit.